app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -54,10 +54,14 @@ class MessageActivity : AppCompatActivity() {
 
         send.setOnClickListener {
             // Easter egg
-            if (content.text.isEmpty()) {
-                presenter.sendBitmap(this, BitmapFactory.decodeResource(resources, R.drawable.mix2))
-            } else {
-                presenter.sendMessage(this, convertToDeviceDataModel())
+            try {
+                if (content.text.isEmpty()) {
+                    presenter.sendBitmap(this, BitmapFactory.decodeResource(resources, R.drawable.mix2))
+                } else {
+                    presenter.sendMessage(this, convertToDeviceDataModel())
+                }
+            } catch (e: Exception) {
```

Comment:
Please do not catch generic exceptions

Reply:
@iamareebjamal could please tell me what else to do instead?